### ISSUE TITLE
Can i get an exist mesh texture with texrecon 

### ISSUE THREAD

**Issue by yang1hu** [GAP-LAB-CUHK-SZ/gaustudio#32]
Hi！Thank for your great project ! I wanna know,If i already have a mesh ! for example ,the mesh come from sugar.if I can get the texture with texrecon  command and what i should do

**Comment by hugoycj** [comment]
Yes, you could modify the render_mesh.py script (https://github.com/GAP-LAB-CUHK-SZ/gaustudio/blob/master/gaustudio/scripts/render_mesh.py) to save the camera pose in the format required by texrecon for texturing. To save the pose correctly, you can follow the approach used in the extract_mesh.py script (https://github.com/GAP-LAB-CUHK-SZ/gaustudio/blob/master/gaustudio/scripts/extract_mesh.py#L115).

For using render_mesh.py scripts, you need the original colmap data folder and running in this way:
```
gs-render-mesh -m <mesh path> -s <colmap dir> -o <output dir>
```

**Comment by yang1hu** [comment]
> Yes, you could modify the render_mesh.py script (https://github.com/GAP-LAB-CUHK-SZ/gaustudio/blob/master/gaustudio/scripts/render_mesh.py) to save the camera pose in the format required by texrecon for texturing. To save the pose correctly, you can follow the approach used in the extract_mesh.py script (https://github.com/GAP-LAB-CUHK-SZ/gaustudio/blob/master/gaustudio/scripts/extract_mesh.py#L115).
> 
> For using render_mesh.py scripts, you need the original colmap data folder and running in this way:
> 
> ```
> gs-render-mesh -m <mesh path> -s <colmap dir> -o <output dir>
> ```

ok！I will try, thank you very mush
